.eslintrc

```diff
@@ -42,8 +42,9 @@
     "linebreak-style": ["error", "unix"],
     "react/no-array-index-key": "off",
     "react/react-in-jsx-scope": "off",
-    "@typescript-eslint/no-var-requires": "off",
-    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/no-var-requires": "error",
```

Comment:
I don't think `require` is bad in some places, and if this enable, need to fix some files